### ISSUE TITLE
Bug in example 7

### ISSUE THREAD

**Issue by jefferis** [natverse/nat.examples#20]
```
> butterfly.brain = insectbraindb_read_brain(species = "Danaus plexippus")
 Error in strsplit(species, " ") : non-character argument 
```

**Comment by jefferis** [comment]
Reported by Enrico Bertolini

**Comment by jefferis** [comment]
Closed by natverse/insectbrainr@455d03a  (See https://github.com/natverse/insectbrainr/issues/1)
